### ISSUE TITLE
Infinite fuel rods from station engineer.

### ISSUE THREAD

**Issue by Seven63** [Regalis11/Barotrauma#4126]
- [Yes] I have searched the issue tracker to check if the issue has already been reported.

**Description**
Stunning the operator of a reactor in a station and stealing their ID card will allow you to access the reactor and take it's fuel rod. once the fuel rod is removed, the operator will place another one in the reactor even if there is not one available or in their inventory. This process can be repeated to collect an (to my knowledge) unlimited number of fuel rods.

**Steps To Reproduce**
1: Start new campaign (I used seed ZphxkCGS).
2: Navigate to the station's reactor room, use a stun gun or stun baton to stun the engineer.
3: handcuff the engineer and take the engineer's ID card and place it in your character's ID card slot.
4: Remove the fuel rod from the station reactor, wait for the engineer to place a new one in.
5: Repeat step 4.

**Version**
0.10.6.2 (Unstable), Windows 10

**Comment by ezjamsen** [comment]
Does the player really need access to station reactors? I'd say probably not. Let's test removing that access.

**Comment by Regalis11** [comment]
@ezjamsen In a sense the players don't have access to the reactors, but it was implemented with ID cards, so it's possible to gain access by stealing a card from one of the NPCs. I guess we could add some kind of a hard-coded restriction as well.

**Comment by ekqvist** [comment]
Added a new editable property "NonPlayerTeamInteractable" for the Item class in Regalis11/Barotrauma-development@252e7e1 and modified code which references the existing property "NonInteractable" so things should properly take into account the new property. I would suggest also testing various ways of assigning orders as I fixed some issues this change caused with them.

Set the new property to true for outpost reactors to prevent characters on player teams from targeting or interacting with them in Regalis11/Barotrauma-development@b3c8cea.

This fix should also address #3970.
